src/hotspot/cpu/aarch64/frame_aarch64.inline.hpp

```diff
@@ -46,7 +46,7 @@ inline frame::frame() {
 
 static int spin;
 
-inline void frame::init(intptr_t* sp, intptr_t* fp, address pc) {
+inline void frame::init(intptr_t* sp, intptr_t* fp, address pc, bool forSignalHandler) {
   assert(pauth_ptr_is_raw(pc), "cannot be signed");
   intptr_t a = intptr_t(sp);
   intptr_t b = intptr_t(fp);
```

Comment:
What are these declarations good for?

Reply:
You mean `bool forSignalHandler`?

Reply:
No, I am referring to lines 51 and 52. These variables are unused as far as I can see.

Reply:
TBH, no idea. The seem to have been added by @theRealAph some 7 years ago ...